site/content/en/docs/tasks/manage/enforce_job_management/setup_default_local_queue.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Setup default LocalQueue"
+date: 2024-12-12
+weight: 10
+description: >
+  Setup default LocalQueue to fullfil a queue label on jobs that submited without queue label.
+---
+
+This page describes how to setup default LocalQueue to ensure that all Workloads submitted to a specific namespace are managed by Kueue,
+even if the `kueue.x-k8s.io/queue-name` label is not specified explicitly.
+
+## Setup default LocalQueue
+
+LocalQueueDefaulting is an Alpha feature that allows the use of a LocalQueue with name `default` as the default LocalQueue
```

Comment:
I would prefer to add the feature banner as here: https://github.com/kubernetes-sigs/kueue/blob/e544dc8a447abfafb471886f9390af0a303b24d5/site/content/en/docs/concepts/topology_aware_scheduling.md?plain=1#L9. Can be a follow up.